### PR TITLE
fix typo and more !

### PR DESCRIPTION
**Pull request by devgdr**
fix typo and more !

### CODE REVIEW

**Review by omidbaharifar — COMMENTED** [review]
This is an educational project, so comments are essential!

**Review by omidbaharifar — COMMENTED** [review]
take a look at this
https://sebhastian.com/javascript-declare-multiple-variables/

File: style.css [hunk 1]
```diff
@@ -17,7 +17,7 @@ body {
     padding: 20px;
     box-shadow: #00000011 0px 0px 15px 5px;
     width: 500px;
-    min-width: fit-content;
+    min-width:500px;
```
Comment (omidbaharifar) [inline style.css:20]: just why?:))

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 <body>
 
     <div class="container">
-        <input type="text" id="text" placeholder="...متن را برای شمارش نقاط وارد کنید">
+        <input dir="auto" type="text" id="text" placeholder="...متن را برای شمارش نقاط وارد کنید">
```
Comment (omidbaharifar) [inline index.html:13]: that was a good one😎

File: main.js [hunk 3]
```diff
@@ -1,52 +1,56 @@
...
 
 inputTag.addEventListener("input", function () {
 
 
-    dotsCount = 0;
+    dotsCount = 0
 
 
-    dotCounter(inputTag.value);
+    dotCounter(inputTag.value)
```
Comment (omidbaharifar) [inline main.js:52]: what is your problem with semocolons dude?😂😔

**Comment by omidbaharifar** [comment]
Thank you for your attention and corrections. 😍

File: main.js (hunk 3 above)
Comment (devgdr) [reply main.js:52]: If u want to use them, you are supposed to use them all over the place, not one here maybe one there
and "commit" was not about semicolons and extra commas anyway!
After all it is an educational project

**Comment by mehdigudy** [comment]
> This is an educational project, so comments are essential!
there is a thing about comments and that is code should be self explanatory enough as your code is here , there is no benefit to comment on a variable assignment or comment on a if statement

**Comment by devgdr** [comment]
> This is an educational project, so comments are essential!

there is a thing about comments and that is code should be self explanatory enough as your code is here , there is no benefit to comment on a variable assignment or comment on a if statement

**Comment by devgdr** [comment]
> take a look at this https://sebhastian.com/javascript-declare-multiple-variables/

thank to the comments couldn't relate damn , still not good for beginners =) but nice one

File: style.css (hunk 1 above)
Comment (devgdr) [reply style.css:20]: because "This is an educational project" XD

**Pull request closed without merging**